pkg/auth/providers/local/local_auth_provider.go

```diff
@@ -469,3 +480,9 @@ func (l *Provider) IsDisabledProvider() (bool, error) {
 func (l *Provider) CleanupResources(*v3.AuthConfig) error {
 	return nil
 }
+
+func userMatchesSearchKey(user *v3.User, searchKey string) bool {
+	return (strings.HasPrefix(user.ObjectMeta.Name, searchKey) ||
+		strings.Contains(strings.ToLower(user.Username), searchKey) ||
+		strings.Contains(strings.ToLower(user.DisplayName), searchKey))
```

Comment:
Should we normalize the whitespace here too?

Reply:
Yeah, good idea.